tdrs-backend/tdpservice/data_files/test/test_api.py

```diff
@@ -134,7 +134,8 @@ def assert_error_report_file_content_matches_without_friendly_names(response):
             + "instructions (linked below) when looking up items and allowable values during the data revision process"
         assert ws.cell(row=8, column=COL_ERROR_MESSAGE).value == ("Every T1 record should have at least one "
                                                                   "corresponding T2 or T3 record with the same "
-                                                                  "RPT_MONTH_YEAR and CASE_NUMBER.")
+                                                                  "Item 4 (reporting month and year) and Item 6 "
```

Comment:
Does Item# help the user?

Reply:
It is apart of the error audit. I can have Miles/Victoria have a look.

Reply:
Yes, the item numbers helps users refer back to the line coding instructions. :)